src/styles/tools/ec-alert.css

```diff
@@ -15,12 +15,37 @@
     }
   }
 
+  .ec-alert__body {
+
+    @media (min-width: 640px) {
```

Comment:
why we use here this breakpoint and why it is not a tailwind breakpoint?

Reply:
640px is the sm tailwind breakpoint, I couldn't use tw directly though as it didn't work, I think we don't have the node-modules for it